blog/2017-12-11-end-to-end-testing-kubernetes-openshift.textile

```diff
@@ -0,0 +1,267 @@
+
+But, with the advent of micro services and container native applications, adding integration and/or
+end-to-end tests to your Kubernetes/OpenShift projects is all the more challenging and cumbersome,
+considering the huge overhead involved in terms of time, cost and resources required in setting up
+the infrastructure.
+
+We at Arquillian community understand this pain, and continuously strive to provide users, a seamless
+experience for writing integration tests with ease by bringing their tests to the real environment and
+deploy new features and projects to production with confidence like never before.
+
+With this blog post, we aim at guiding you in writing integration tests for your deployments done on
+Kubernetes or OpenShift clusters using just "Arquillian Kube":https://github.com/arquillian/arquillian-kube for applications supported with resource
+descriptors in the form of Kubernetes/OpenShift manifest or template files.
+
+Moving a step further, we also demonstrate building and deploying from scratch, applications with no
+deployment configuration, leveraging the power of "Fabric8 Maven Plugin":https://maven.fabric8.io/ along with the Arquillian Kube
+Extension.
+
+*If, you wish to read no more and see it all in action, head straight to our specially curated
+examples to help you get started with ease.*
+
+h3. Example 1
+
+Deploying a sample PHP Guestbook application with Redis on Kubernetes from the resource descriptor
+manifest file and testing it using Arquillian Kube Extension for Kubernetes and Kubernetes custom assertions.
+
+Source: "arquillian-testing-microservices/kubernetes-deployment-testing":https://github.com/arquillian-testing-microservices/kubernetes-deployment-testing
+
+h3. Example 2
+
+Deploying a Wordpress and My SQL application to OpenShift from a Template file and testing it using Arquillian
+Kube Extension for OpenShift and  Fabric8 OpenShift Client.
+
+Source: "arquillian-testing-microservices/openshift-deployment-testing":https://github.com/arquillian-testing-microservices/openshift-deployment-testing
+
+h3. Example 3
+
+Building and deploying a sample SpringBoot GuestBook application with zero deployment configuration using
+"Fabric8 Maven Plugin":https://maven.fabric8.io/ and "Arquillian Kube Extension":https://github.com/arquillian/arquillian-kube .
+
+Fabric8 Maven Plugin aids in building Docker images and creating Kubernetes and OpenShift resource
+descriptors for the application that allows for a quick ramp-up with some opinionated defaults and Arquillian
+Kube Extension deploys the application from the generated resource descriptors and then executes deployment tests.
+
+Source: "arquillian-testing-microservices/zero-config-deployment-test":https://github.com/arquillian-testing-microservices/zero-config-deployment-test
+
+*For a more in-depth step by step coverage of deployment testing, continue reading below.*
+
+*Step 1: Building Docker Images and creating Resource Descriptors.*
```

Comment:
This Step 1 is a really concrete example of using f8 maven plugin to generate resources, since Example 3 is about this concrete case, what I would do is just remove this step since it is not mandatory.